modules/shared/src/main/scala/org/tessellation/security/signature/SignedValidator.scala

```diff
@@ -66,6 +73,22 @@ object SignedValidator {
       else
         NotEnoughSignatures(signedBlock.proofs.size, minSignatureCount).invalidNec
 
+    def validateSignatureExclusiveForAddress[A <: AnyRef](
```

Comment:
maybe just `isSignedBy` or `isSignedExclusivelyBy` ?

Reply:
`isSignedBy` and `isSignedExclusivelyBy` takes `Id`, not `Address`. I can prepare version of theses methods which take `Address` if you want, but I think above is good solution.

Reply:
`exclusive` -> `exclusively` and what's wrong with such a function?
```
def isSignedBy(address: Address)(signed: Signed[A]): F[SignedValidationErrorOr[Signed[A]]] = ???
```

Reply:
Oh, I misunderstood, I thought you were refereing to `isSignedBy` and `isSignedExclusivelyBy` methods from `SignedOps`.

Reply:
Changed.